x-pack/legacy/plugins/reporting/export_types/printable_pdf/server/execute_job/compatibility_shim.js

```diff
@@ -4,41 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import url from 'url';
-import { getAbsoluteUrlFactory } from '../../../../common/get_absolute_url';
-import { i18n } from '@kbn/i18n';
-
-export function compatibilityShimFactory(server) {
-  const getAbsoluteUrl = getAbsoluteUrlFactory(server);
-
-  const getSavedObjectAbsoluteUrl = (job, savedObject) => {
```

Comment:
See also: the `create_job` compatibility shim, which is now the only place that parses saved objects by ID into Kibana URLs

Reply:
Correction: this wasn't actually parsing saved objects by ID, but it was handling a structure of data that isn't relevant to the `create_job` outputs for PDF and PNG.

(worth double checking that)